etc/kayobe/kolla.yml

```diff
@@ -456,7 +456,7 @@ kolla_build_args:
 #kolla_enable_ceilometer_horizon_policy_file:
 #kolla_enable_ceilometer_ipmi:
 #kolla_enable_cells:
-#kolla_enable_central_logging:
+kolla_enable_central_logging: true
```

Comment:
Undecided on whether we should set these flags by default, or just provide config & allow users to enable when they want to. I suppose we will want them enabled in more cases than not.

Reply:
As discussed in infra R&D meeting, this will be correct in more cases than not, so let's keep it.